DOCS-9525 Update who has access to public portals. (#137)
mulesoft/docs-exchange: modules/ROOT/pages/about-portals.adoc

endif::[]
:imagesdir: ../assets/images/

> Exchange public portals are accessible anyone using the internet, enabling developers outside of an organization to access that organization's REST, SOAP, and HTTP APIs. This means users who are not log in can download public portals.

Reviewer: ```suggestion
Exchange public portals are accessible for anyone using the internet, enabling developers outside of an organization to access that organization's REST, SOAP, and HTTP APIs. Users who are not logged in can download assets from public portals.
```
